pyproject.toml

```diff
@@ -37,6 +37,7 @@ unfixable = []
 "_init.py" = ["F401", "F403"]  # unused import
 "taipy/config/stubs/pyi_header.py" = ["F401", "F403"]  # unused import
 "taipy/templates/*" = ["F401", "F403", "T201"]  # unused import, `print` found
+"taipy/gui/utils/types.py" = ["B024"] # abstract base class with no abstract methods
```

Comment:
I believe Giang proposed another fix for that in a [PR](https://github.com/Avaiga/taipy/pull/1259/files#diff-d9fb688584a1df48124e03582294982435183d2b279b628e5e2bcbb4489913e5)

Is it the same issue?

Reply:
Yes I believe this is redundant now